.github/utils/tutorials.sh

```diff
@@ -73,15 +83,38 @@ for script in $scripts_to_run; do
     fi
 
     if [[ "$script" == *".py" ]]; then
-        time python $script
+        output=$(time python $script)
     else
-        sudo $python_path/bin/ipython -c "%run $script"
+        output=$(sudo $python_path/bin/ipython -c "%run $script")
     fi
 
-    if [ ! $? -eq 0 ]; then
+    echo $output > $script-output.txt
+    if [ $? -eq 0 ]; then
+        echo "Execution completed successfully. See artifacts for the tutorial's output."
+    else
+        echo "===================================================="
+        echo "|  $script FAILED!"
+        echo "===================================================="
+        echo "Output of the execution: "
+        echo $output
         failed=$failed" "$script
     fi
 
+    # Restart the containers
+    if [[ "$make_python_path_editable" == "RESTART" ]]; then
+        docker stop elasticsearch
+        docker stop tika
+        docker stop graphdb
+
+        docker rm elasticsearch
+        docker rm tika
+        docker rm graphdb
```

Comment:
graphdb is not used in the tutorials anymore because we now use the InMemoryKnowledgeGraph instead. So we don't need to run the graphdb container at all.